docs/operation/cluster_and_service_configuration.rst

```diff
@@ -498,6 +510,52 @@ To change the password of an existing user, perform the following operations:
       - Configuring LDAP
       - Configuring Active Directory(AD)
 
+Loki Configuration Customization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Default configuration for Loki can be overridden by editing its
+Cluster and Service ConfigMap ``metalk8s-loki-config`` in namespace
+``metalk8s-logging`` under the key ``data.config.yaml``:
+
+.. code-block:: shell
+
+    root@bootstrap $ kubectl --kubeconfig /etc/kubernetes/admin.conf \
+                       edit configmap -n metalk8s-logging \
+                       metalk8s-loki-config
+
+The following documentation is not exhaustive and is just here to give
+some hints on basic usage, for more details or advanced
+configuration, see the official `Loki documentation`_.
+
+.. _Loki documentation: https://grafana.com/docs/loki/latest/configuration/
+
+Change the logs retention period
+""""""""""""""""""""""""""""""""
+
+Retention period is the time the logs will be stored and available before
+getting purged by the table manager.
```

Comment:
```suggestion
Retention period is the time the logs will be stored and available before
getting purged by the Table Manager.
```
Also since you mentioned **Table Manager** which is a Loki component, I think it is worth adding a definition for it in our glossary or reference directly the Loki doc.

What do you think?

Reply:
I'm not sure, maybe I should rather remove the mention to table manager, it's a Loki implementation detail.
It does not bring anything to talk about it here, apart from confusing the reader.